SPECS/kata-containers/kata-containers.spec

```diff
@@ -57,6 +57,7 @@ Patch2:         0003-Merged-PR-9805-Add-support-for-MSHV.patch
 Patch3:         0004-Merged-PR-9806-Fix-enable_debug-for-hypervisor.clh.patch
 Patch4:         0005-Merged-PR-9956-shim-avoid-memory-hotplug-timeout.patch
 Patch9:         runtime-reduce-uvm-high-mem-footprint.patch
+Patch10:        drop-mut-for-variables-tahat-are-not-mutated.patch
```

Comment:
potential to fix 'tahat' to 'that'. Please, why is there patch9 and path10 and not e.g. patch5? Not blocking here, happy to move on

Reply:
'tahat' also appears in the kata-containers-cc patch

Reply:
Falak did a quick turn around on those, I can tidy that up.

Reply:
As for patch numbers, maybe there were previously patches 5-8 that have since been removed? I'll condense the numbering while I'm here.